pages/pas_reset_page.py

```diff
@@ -0,0 +1,33 @@
+from selenium.webdriver.common.by import By
+from pages.base_page import BasePage
+
+
+class ResetPassword(BasePage):
+
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.open("/web/index.php/auth/requestPasswordResetCode")
```

Comment:
If we could navigate to page by link it could be separate method for it. 
Go test test_password_reset_submit step by step, you perform several unnecessary navigations